src/demo/_theme.sass

```diff
@@ -0,0 +1,173 @@
+  line-height: 1
+  margin: 0
+  padding: 8px 8px 4px 8px
+  position: absolute
+  text-align: left
+  top: 0
+
+.ca11yndar__month-display
+  display: block
+  font-size: 32px
+  margin-bottom: 4px
+  text-transform: uppercase
+
+.ca11yndar__year-display
+  margin-left: 2px
+
+.ca11yndar__day-names
+  th
+    color: lighten(black, 75%)
+    font-size: 10px
+    font-weight: normal
+    height: 30px
+    width: 30px
+
+.ca11yndar__toggle
+  background-color: white
+  border-radius: 3px
+  border: 1px solid $grey-90
+  font-size: 16px
+  height: 38px
+  margin: 0
+  outline: none
+  padding: 0 8px
+
+  &:hover,
+  &:focus
+    color: $highlight
+
+  &:focus
+    border-color: $highlight
+
+  > svg
+    display: block
+    height: 24px
+    width: 24px
+
+.ca11yndar__nav
+  background-color: $grey-90
+  border-radius: 100px
+  border: none
+  bottom: 14px
+  color: inherit
+  display: block
+  font-weight: bold
+  height: 24px
+  margin: 0
+  outline: none
+  padding: 4px
+  position: absolute
+  transition: all .2s ease-in-out
+  width: 24px
+
+  svg
+    display: block
+    height: 100%
+    width: 100%
+
+  &:hover,
+  &:focus
+    background-color: $highlight
+    color: #fff
+
+  &.-next
+    left: 40px
+
+  &.-prev
+    left: 8px
+
+.ca11yndar__picker
+  background-color: #fff
+  background-image: linear-gradient(15deg, #fff, 90%, rgba($highlight, 0.05))
+  border-radius: 3px
+  border: 1px solid $grey-90
+  box-shadow: 2px 4px 0px rgba(black, 0.02)
+  color: $textColor
+  font-size: 11px
+  font-weight: bold
+  margin-top: 16px
+  overflow: hidden
+  padding: 8px 8px 8px 90px
+  position: absolute
+  right: 0
+  top: 100%
+  transition: all .2s ease-in-out
+  z-index: 1
+
+  *
+    -khtml-user-select: none
+    -moz-user-select: -moz-none
+    -o-user-select: none
+    -webkit-user-select: none
+    user-select: none
+
+  &[aria-hidden="true"]
+    opacity: 0
+    transform-origin: 100% 0%
+    transform: translate3d(24px, -64px, 0) scale(0.5)
+    visibility: hidden
+
+  &:hover
+    .ca11yndar__day
+      &:hover,
+      &:hover:focus
+        background-color: rgba($highlight, 0.5)
+
+      &:focus
+        background-color: transparent
+        color: inherit
+
+.ca11yndar__table
+  border-collapse: collapse
+  border-spacing: 0
+
+.ca11yndar__row
+  &:last-child .ca11yndar__cell
+    border-bottom: none
+
+.ca11yndar__cell
+  padding: 0
+  position: relative
+
+  &:first-child
+    border-left: none
+
+  &:last-child
+    border-right: none
+
+.ca11yndar__day
+  appearance: none
+  background: transparent
+  border-radius: 32px
+  border: 2px solid transparent
+  color: inherit
+  display: block
+  font-size: 11px
+  font-weight: bold
+  height: 32px
+  outline: none
+  transition: all .2s ease-in-out
+  width: 32px
+
+  &.-today
+    color: $highlight
+
+  &.-selected
+    background-color: $highlight
+    color: #fff
+
+  &:hover
+    background-color: rgba($highlight, 0.5)
+
+  &:focus
+    border-color: rgba($highlight, 0.5)
```

Comment:
As you translate this into an npm module, you could put all of the sass into one mixin, so the css only exists when they call the mixin, with parameters for your 3 colors (text, highlight, and gray).